user/client.go

```diff
@@ -588,3 +588,20 @@ func (c *Client) DeleteExternalAccount(ctx context.Context, params *DeleteExtern
 	err = c.Backend.Call(ctx, req, resource)
 	return resource, err
 }
+
+type PasswordCompromisedParams struct {
+	clerk.APIParams
+	UserID string `json:"user_id"`
+}
+
+// PasswordCompromised marks the user's password as compromised.
+func (c *Client) PasswordCompromised(ctx context.Context, params *PasswordCompromisedParams) (*clerk.User, error) {
```

Comment:
🔧 maybe we don't need a params struct here and we can pass the userID directly as a standalone parameter

```suggestion
func (c *Client) PasswordCompromised(ctx context.Context, userID string) (*clerk.User, error) {
```